examples/magento-graphcms/components/GraphCMS/Conditions.graphql

```diff
@@ -0,0 +1,29 @@
+fragment ConditionNumber on ConditionNumber {
+  __typename
+  property
+  operator
+  numberValue: value
+}
+
+fragment ConditionText on ConditionText {
+  __typename
+  property
+  stringValue: value
+}
+
+fragment ConditionOr on ConditionOr {
+  __typename
+  conditions {
+    ...ConditionNumber
+    ...ConditionText
+  }
+}
+
+fragment ConditionAnd on ConditionAnd {
+  __typename
+  conditions {
+    ...ConditionNumber
+    ...ConditionText
+    ...ConditionOr
+  }
+}
```

Comment:
This query can be moved to the graphcms-ui package since we can assume these Conditions exist.